scripts/fetch-github-data.js

```diff
@@ -245,6 +245,10 @@ const createRepoDataWithResponse = (json, monorepo) => {
     if (json.topics.length === 0) {
       json.topics = packageJson.keywords;
     }
+
+    if (!json.licenseInfo || (json.licenseInfo && json.licenseInfo.key === 'other')) {
+      json.licenseInfo = getLicenseFromPackageJson(packageJson);
```

Comment:
Would you need to do a check here to make sure the license doesn't get set to `undefined` e.g. in the case where `key = 'other'` and the license is missing from the package.json?

Something like:
```suggestion
      json.licenseInfo = getLicenseFromPackageJson(packageJson) || json.licenseInfo;
```

Reply:
Good point, thank you for the suggestion! It probably won't happen often, but it's good to have a fallback on `package.json` read result.